portfolio/src/main/webapp/index.html

```diff
@@ -33,6 +33,9 @@ <h1 style="text-align:center"> Sean Tisdale Portfolio </h1>
             <p>Click here to get a random message:</p>
             <button onclick="addRandomFact()">Facts</button>
             <div id="fact-container"></div>
+            <br>
+            <button onclick="getHelloArrowFunctions() ">Hello</button>
```

Comment:
hmm, did the walkthrough request that you name this `-ArrowFunctions`?

getHello() is probably a better name, since it reflects what the function tries to do instead of the implementation details (in this case, ArrowFunctions just refers to the => syntactic sugar used to create an anonymous function object)

Reply:
solved